### ISSUE TITLE
support tagged links with non-space text immediately trailing closing `]`

### ISSUE THREAD

**Issue by phrohdoh** [athensresearch/athens#2000]
Often, in Roam, I will write something like:
```
used #[[babashka]]'s XYZ feature with ABC in order to QWE
```

For this to work in Athens (I am using `2.0.0-beta.6`) as it does in Roam, which is to render `#babashka's` with the `#babashka` as a link, I must put a space between the `#[[babashka]]` and the `'s`:

```
used #[[babashka]] 's XYZ ...
```

---

nothing critical, but feels like a [paper cut](https://en.wikipedia.org/wiki/Paper_cut_bug) to me

**Comment by alexandergunnarson** [comment]
Yup, I've been encountering this too today. The referencing happens properly, but the UI doesn't reflect the parsing.
